README.rst

```diff
@@ -552,15 +552,16 @@ Each queue can have tasks in the following states:
 - ``scheduled``: Tasks that are scheduled for later execution.
 - ``error``: Tasks that failed with an error.
 
-To get a list of all tasks for a given queue and state, use
-``Task.tasks_from_queue``. The method gives you back a tuple containing the
-total number of tasks in the queue (useful if the tasks are truncated) and a
-list of tasks in the queue, latest first. Using the ``skip`` and ``limit``
-keyword arguments, you can fetch arbitrary slices of the queue. If you know the
-task ID, you can fetch a given task using ``Task.from_id``. Both methods let
-you load tracebacks from failed task executions using the ``load_executions``
-keyword argument, which accepts an integer indicating how many executions
-should be loaded.
+To get a count of the number of tasks for a given queue and state, use
+``Task.count_tasks_from_queue``. To get a list of all tasks for a given queue
```

Comment:
This is actually called `task_count_from_queue` in the code.

Also, another approach would be to let you pass 0 into `tasks_from_queue` (currently it's undefined). That way we wouldn't need to add a new function.

Reply:
`tasks_from_queue` uses `mget` and then loops over all the returned tasks. It's much faster to use `zcount` to only get the count of tasks from redis.

Reply:
Fixed the readme with 843e0165f2379780a583b438bc7fdb10498851ad.

Reply:
I meant that passing `limit=0` should make it just return the count and an empty list.

Reply:
Oh, got it. I prefer methods to not change behavior. Would you be ok with leaving then separate methods?